bpython/test/test_autocomplete.py

```diff
@@ -245,12 +245,12 @@ def test_att_matches_found_on_old_style_instance(self):
                                              locals_={'a': OldStyleFoo()}),
                             set(['a.method', 'a.a', 'a.b']))
         self.assertIn(u'a.__dict__',
-                      self.com.matches(3, 'a._', locals_={'a': OldStyleFoo()}))
+                      self.com.matches(3, 'a.__', locals_={'a': OldStyleFoo()}))
```

Comment:
`3` and `a._` means to test for `a._<Tab>` while `3` and `a.__` means to test for `a._<Tab>_` which is a different test. The first argument is the position of the cursor at the time of the autocompletion call. So leaving the first value set to `3` does not preserve the meaning of the test.